packages/arb-token-bridge-ui/tests/support/commands.js

```diff
@@ -29,10 +29,9 @@ Cypress.Commands.add("login", () => {
   cy.visit(`/`);
   cy.findByText("Agree to terms").should("be.visible").click();
   cy.findByText("MetaMask").should("be.visible");
-  cy.findByText("Connect to your MetaMask Wallet")
-    .click()
-    .then(() => {
-      cy.acceptMetamaskAccess().should("be.true");
-      cy.switchToCypressWindow().should("be.true");
-    });
-});
\ No newline at end of file
+  cy.findByText("Connect to your MetaMask Wallet").click();
+  cy.acceptMetamaskAccess().then(connected => {
```

Comment:
@spsjvc When run locally on Windows, the test passed as expected but when I run it on codespaces, it failed. Could you please let me know if it works on Mac? I'm expecting it should work and it should be a codespaces-specific issue. Filed a ticket at synpress: https://github.com/Synthetixio/synpress/issues/498

Reply:
Works!